ShiftLogger.Mefdev/ShiftLoggerUI/Controllers/WorkerShiftBase.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Spectre.Console;
+
+namespace ShiftLogger.Mefdev.ShiftLoggerUI.Controllers
+{
+	public class WorkerShiftBase
```

Comment:
🟠 **Class Name**

💡 `WorkerShiftBase` doesn't really convey what this class does. This seems like a UI display/writer class. Maybe some re-design is required.